frontend/tasks/pages.js

```diff
@@ -201,16 +201,13 @@ function generateStaticPage(prepareForClient, pageName, pageParam, component, {
     <meta name="availableLanguages" content={ available_locales } />
     <meta name="viewport" content="width=device-width" />
 
-    <link rel="localization" href="/static/locales/{locale}/app.ftl" />
```

Comment:
Still haven't quite gotten through skimming this stuff to the point that I understand how localizations are loaded. But, I wonder if we're loading them dynamically in JS, we can possibly solve #2229 by hashing the resource URL filenames and loading the right ones to bust cache? Maybe need a manifest of locale -> hashed URL